dev.hdos.HDOS.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally this, the icon and the desktop files should be maintained "on your end", but this is not a blocker.

Reply:
You would like them versioned?

Reply:
It's a deliverable like the JAR file is. If you are already creating and uploading a release elsewhere, it makes more sense to maintain it there.